### ISSUE TITLE
Robot Framework standard libraries are not getting imported

### ISSUE THREAD

**Issue by udhayasankari** [millennialmedia/intellibot#124]
Hi , 

In my robot test file as below , am getting import not found error. Though path is updated for library paths. it works with pybot but with pycharm intellibot it says import not found.

Library           Rammbock
Library           OperatingSystem
Library           Collections

Python 2.7.10 (default, May 23 2015, 09:40:32) [MSC v.1500 32 bit (Intel)] on win32
>>> sys.path
['C:\\Program Files (x86)\\JetBrains\\PyCharm Community Edition 5.0.1\\helpers\\pydev', 'D:\\WIFI\\NWIFI\\Automation\\trunk', 'C:\\Program Files (x86)\\JetBrains\\PyCharm Community Edition 5.0.1\\helpers\\pydev', 'C:\\WINDOWS\\system32\\python27.zip', 'C:\\Python27\\DLLs', 'C:\\Python27\\lib', 'C:\\Python27\\lib\\plat-win', 'C:\\Python27\\lib\\lib-tk', 'C:\\Python27', 'C:\\Python27\\lib\\site-packages', 'D:\\WIFI\\NWIFI\\Automation\\trunk']

**Comment by mtrubs** [comment]
if it works with pybot then it is likely a matter of how you have the python interpretor setup in your project. let me poke in pycharm and get back to you.

**Comment by mtrubs** [comment]
I updated the wiki page to include PyCharm notes.

https://github.com/millennialmedia/intellibot/wiki/Python-Interpreter

I suspect that your IDE is not using the version of python you think it is using or nothing is set.  Mine was empty when I first opened the project.

Let me know if this helps.

**Comment by udhayasankari** [comment]
I did gave the python-Interpreter pointing to python installed in my machine. 
I am using windows 7 . 

Still facing same problem. Infact even for normal python file , import * says file not found. So I guess, system path is not loaded /; taken correctly.

But python console opened through pycharm displays correct path. as below

>>> sys.path
['C:\\Program Files (x86)\\JetBrains\\PyCharm Community Edition 5.0.1\\helpers\\pydev', 'D:\\WIFI\\NWIFI\\Automation\\trunk', 'C:\\Program Files (x86)\\JetBrains\\PyCharm Community Edition 5.0.1\\helpers\\pydev', 'C:\\WINDOWS\\system32\\python27.zip', 'C:\\Python27\\DLLs', 'C:\\Python27\\lib', 'C:\\Python27\\lib\\plat-win', 'C:\\Python27\\lib\\lib-tk', 'C:\\Python27', 'C:\\Python27\\lib\\site-packages', 'D:\\WIFI\\NWIFI\\Automation\\trunk']

**Comment by udhayasankari** [comment]
Hi , 

Update to this , I tried virtual env and it worked. 
Not sure why direct python path is not working.

**Comment by ovkhasch** [comment]
You can also try to set the paths to your libraries in PYTHONPATH env variable instead of PATH.

**Comment by tbrown9164** [comment]
I came across something like this when I had migrated a robotframework env to a new machine.   The wrong libraries (the python ones) only had been pip'd (my fault, I was on vacation for a while!).  For example:  Pypi 'selenium' was pip'd and looked fine in the IDE (Pycharm) module listing,  however what was really needed was the pip of the 'robot framework selenium'  which is  'robotframework-selenium',  not just 'selenium' for native python use.     So everything appeared ok at first glance, and all of the builtin keywords were fine,  but no added std libraries (e.g. selenium) or any of their keywords worked.   The 'pip list' comparison between the legacy and migrated to machine gave me the hint.  Give that a look at at least.
